src/clusterfuzz/_internal/tests/core/bot/tasks/trials_test.py

```diff
@@ -23,7 +24,7 @@
 
 
 @test_utils.with_cloud_emulators('datastore')
-class TrialsTest(unittest.TestCase):
+class TrialsTest(fake_filesystem_unittest.TestCase):
```

Comment:
nit: Maybe add one test with broken json thats expected to fail?

Reply:
done, however a broken JSON does not fail, it just logs the exception and continues the execution. The test that was added checks that a broken JSON does not override existing values or have any effect.